get_kserve_manifests.sh

```
@@ -0,0 +1,56 @@
+  dest=$(yq e ".map.\"${component}\".dest" "${CONFIG_FILE}")
+  local_flag=$(yq e ".map.\"${component}\".local // false" "${CONFIG_FILE}")
+  dest_dir="${DEST_BASE}/${dest}"
+  mkdir -p "${dest_dir}"
+
+  if [[ "${local_flag}" == "true" ]]; then
+    echo "Collecting ${component} from local ${src}/"
+    cp -r "${src}/." "${dest_dir}/"
+    rm -rf "${dest_dir}/kserve-module"
+  else
+    git_url=$(yq e ".map.\"${component}\".\"git.url\"" "${CONFIG_FILE}")
+    ref_type=$(yq e ".map.\"${component}\".ref_type // \"branch\"" "${CONFIG_FILE}")
```

Shellcheck (0.11.0)

[warning] ref_type appears unused. Verify use (or export if used externally).

(SC2034)